pkg/codec/vaapi/vp8.go

```diff
@@ -545,6 +547,10 @@ func (e *encoderVP8) Controller() codec.EncoderController {
 	return e
 }
 
+func (e *encoderVP8) ForceKeyFrame() {
```

Comment:
This interface has been replaced by `func Controller() codec.EncoderController`
https://github.com/pion/mediadevices/pull/338/files#diff-f1af839aa40e8d1553814897ef277f31f625ed6daf6e6f44bfeb950905efabb8R122-R146

Reply:
I don't understand, this function method already exists and returns the encoder itself.

Reply:
I also don't remember what I wanted to point and old CI log is gone.
The interface seems be correct.